serverlessworkflow/modules/ROOT/pages/service-orchestration/orchestration-of-openapi-based-services.adoc

```diff
@@ -0,0 +1,423 @@
+      properties:
+        leftElement:
+          format: float
+          type: number
+        rightElement:
+          format: float
+          type: number
+----
+
+<1> The `operationId` of the REST operation
+<2> The data structure of the REST operation
+
+Use the `operationId` to compose the final URI in the function definition as shown in the example:
+
+.OpenAPI Functions definition in the Temperature Conversion example
+[source,json]
+----
+{
+   "functions": [
+    {
+      "name": "multiplication",
+      "operation": "specs/multiplication.yaml#doOperation" <1>
+    },
+    {
+      "name": "subtraction",
+      "operation": "specs/subtraction.yaml#doOperation" <2>
+    }
+  ]
+}
+----
+
+<1> The OpenAPI function definition of the multiplication function
+<2> The OpenAPI function definition of the subtraction function
+
+=== Acessing the Functions in the Workflow States
+
+The next step is to use the function definitions in the workflow states. In the example application, the functions are called sequentially in an link:{spec_doc_url}#operation-state[Operation State]:
+
+[source,json]
+----
+{
+   "states": [
+    {
+      "name": "SetConstants",
+      "type": "inject",
+      "data": {
+        "subtractValue": 32.0,
+        "multiplyValue": 0.5556
+      },
+      "transition": "Computation"
+    },
+    {
+      "name": "Computation",
+      "actionMode": "sequential",
+      "type": "operation", <1>
+      "actions": [
+        {
+          "name": "subtract",
+          "functionRef": {
+            "refName": "subtraction", <2> 
+            "arguments": { <3>
+              "leftElement": ".fahrenheit", 
+              "rightElement": ".subtractValue" 
+            }
+          }
+        },
+        {
+          "name": "multiply",
+          "functionRef": {
+            "refName": "multiplication", <4> 
+            "arguments": { <5> 
+               "leftElement": ".difference",
+               "rightElement": ".multiplyValue"
+            }
+          }
+        }
+      ],
+      "end": {
+        "terminate": "true"
+      }
+    }
+  ]
+}
+----
+
+<1> The Operation State definition
+<2> Reference to the `subtraction` function
+<3> Definition of the `subtraction` function arguments
+<4> Reference to the `multiplication` function
+<5> Definition of the `multiplication` function arguments
+
+Any link:{spec_doc_url}#Action-Definition[workflow actions] can call a function definition, having a similar way of referencing the functions that you previously defined in the workflow.
+
+To map the function's arguments, refer to the parameters described in the OpenAPI specification file, link:{open_api_spec_url}#operation-object[operations section]. In this example, the parameters are part of the request body.
+
+The Kogito engine will map the parameter naming according to the OpenAPI specification. Then, you can use {understanding-jq-expressions}[JQ expressions] to extract the payload data and map it to the required parameters.
```

Comment:
I would move this up a bit, to line 184, just after the first sentence.  This way you are first "explaining the theory" fully and then showing the example.